src/components/addBook.js

```diff
@@ -31,8 +32,8 @@ const AddBook = () => {
     <div className="bookInput">
       <h3 className="addBook">ADD NEW BOOK</h3>
       <form className="bookForm" onSubmit={submitHandler}>
-        <input type="text" className="title" name="title" placeholder="Book title" />
-        <input type="text" className="author" name="author" placeholder="Author" />
+        <input type="text" className="inputTitle" name="title" placeholder="Book title" />
+        <input type="text" className="inputAuthor" name="author" placeholder="Author" />
         <button type="submit" className="btnAdd">ADD BOOK</button>
       </form>
```

Comment:
- [ ] It would be nice if you could style the input fields and the Add Book button to span across to the right so everything aligns and match the original design. Please check the guide from the screenshot to better compare and understand what needs to be done. 👍🏽

**Your Design**

<img width="1721" alt="Screenshot 2023-05-13 at 3 43 12 PM" src="https://github.com/anbehindY/book-store/assets/50941074/8487694c-ca85-4d4a-a864-d84075151d62">


**The Original Design**

<img width="1264" alt="Screenshot 2023-05-13 at 3 45 42 PM" src="https://github.com/anbehindY/book-store/assets/50941074/acba6d35-6e43-4588-880a-8bfcdb1e2ee3">